etc/skeleton/.travis.yml

```diff
@@ -1,13 +1,13 @@
 language: go
 
 go:
-  - '1.10'
+  - '1.11'
```

Comment:
Should we also continue to test 1.10? I don't know how long we promise compatibility.

Reply:
Officially https://github.com/src-d/guide/blob/master/engineering/conventions/go.md#supported-go-versions

But I would suggest deciding on the consistent strategy to get there: right now it's `1.10` basically _everywhere_ in bblfsh. 

So for the purpose of trackable update if I may, I would suggest having an ☂️ issue, listing all the places we want to update the version/make sure it complies with policy and doing that in a single run?

That is, in case we decide that it's a priority ofcourse

Reply:
This is for the driver only, not SDK. So I think it may be okay to support a single version. Most users will see the docker image anyway.